core/snippets/README.md

```diff
@@ -7,7 +7,15 @@ Custom format to represent snippets.
 - Custom file ending `.snippet`.
 - Supports syntax highlighting in VSCode via an [extension](https://marketplace.visualstudio.com/items?itemName=AndreasArvidsson.andreas-talon)
 - Supports auto-formatting in VSCode via an [extension](https://marketplace.visualstudio.com/items?itemName=AndreasArvidsson.andreas-talon)
-- Support for insertion and wrapper snippets. Note that while the snippet file syntax here supports wrapper snippets, we still need to add the proper voice commands; stay tuned.
+- Support for insertion and wrapper snippets. Note that while the snippet file syntax here supports wrapper snippets, you will need to install [Cursorless](https://www.cursorless.org/) for wrapper snippets to work. You'll also need to add the following line to your `settings.talon` file:
+
+  ```talon
+  tag(): user.cursorless_use_community_snippets
+  ```
```

Comment:
This isn't really a problem for the average user, but for Cursorless maintainers, they'll need to enable this tag in a `mode: command` context. Otherwise the local Cursorless Talon spoken form tests will break, because it's not possible to deactivate a tag. Any thoughts on how to handle that @AndreasArvidsson?

Reply:
Where we're actually using this we could just put mode command?

Reply:
update from meet-up:

- let's just leave this as is, and when Cursorless developers run into local talon test failures, we tell them to move the `tag(): user.cursorless_use_community_snippets` to a file with `mode: command` at the top